source/postprocess/depth_average.cc

```diff
@@ -463,6 +463,10 @@ namespace aspect
 
             if ( output_all_variables || std::find( output_variables.begin(), output_variables.end(), "vertical mass flux") != output_variables.end() )
               variables.push_back("vertical_mass_flux");
+
+            if ( output_all_variables || std::find( output_variables.begin(), output_variables.end(), "composition mass") != output_variables.end() )
```

Comment:
I guess the position of these lines is a trade-off between keeping similar quantities together in the output (C_1 C_1_mass) and breaking compatibility of postprocessing scripts by changing the order of existing columns?

Reply:
Yes.